doc/sheet_functions.md

```diff
@@ -279,6 +285,22 @@ ETH spent in the last 7 days, ETH spent in the last day.
 **Kind**: global function  
 **Returns**: <code>Array</code> - of all projects.  
 **Customfunction**:   
+<a name="SAN_BITMEX_PERPETUAL_CONTRACT_FUNDING_RATE"></a>
+
+## SAN\_BITMEX\_PERPETUAL\_CONTRACT\_FUNDING\_RATE(projectSlug, from, to, interval) ⇒ <code>Array</code>
+Returns the funding rate that is paid by one side of the perpetual contract to the other.
+
+**Kind**: global function  
+**Returns**: <code>Array</code> - of the funding rate that is paid by one side of the perpetual contract to the other.  
```

Comment:
Should it be plural?
 funding rates that are paid by one of the sides of the perpetual contract to the other